source/code/include/playfab/PlayFabPlatformMacros.h

```diff
@@ -29,4 +29,22 @@
 // Hence to specify only Windows, we have check for ! _DURANGO.
 #if defined(_WIN32) && !defined(_DURANGO)
 #define PLAYFAB_PLATFORM_WINDOWS
-#endif //_WIN32
\ No newline at end of file
+#endif //_WIN32
+
+#if defined(PLAYFAB_PLATFORM_WINDOWS) || defined(PLAYFAB_PLATFORM_XBOX)
+typedef signed __int64 Int64;
+typedef signed __int32 Int32;
+typedef signed __int16 Int16;
+
+typedef unsigned __int64 Uint64;
+typedef unsigned __int32 Uint32;
+typedef unsigned __int16 Uint16;
```

Comment:
why is this typedef necessary? windows and xbox have access to the stdint.h header so they can use types like uint64_t

this has bothered me before when looking at sdk code. it'd be great to get rid of these typedefs if possible

Reply:
I didn't define it, I just moved it to the platform-specific file, because it clearly is.
I think the intent is to have a consistent label for the strict-types, and they aren't platform consistent.
I'm willing to improve this, but not in this PR.